Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -53,7 +53,7 @@ def audit_desc(type, name, desc_call)
         desc_problem "Description shouldn't start with an article." if regex_match_group(desc, /^(the|an?)(?=\s)/i)
 
         # Check if invalid lowercase words are at the start of a desc.
-        if !VALID_LOWERCASE_WORDS.include?(string_content(desc).split.first) &&
+        if !VALID_LOWERCASE_WORDS.include?(string_content(desc).split.first) && # rubocop:disable Style/InverseMethods (false positive)
```

Comment:
> Is it a false positive? Is it not just because we don't have that part of ActiveSupport required for cops and we've not excluded these files from this check like we already do with Rails?

Yes to both, I consider the latter to be a false positive (meaning "this code is correct, rubocop should be ignored" not "rubocop has a point, but we've chosen to do something else"), but i'm open to a different wording. What do you suggest?

> Edit: Oh wait I see - they've taken a native RuboCop check and extended it... To be honest I don't understand why they've done that but also have Rails/NegateInclude - is there any difference? It is possible to disable the ActiveSupport parts of Style/InverseMethods

I don't think there's a difference. I've filed https://github.com/rubocop/rubocop-rails/issues/941 in the meantime. I describe a couple solutions there, I think I'd prefer the inline disable here in the meantime. Let me know what you'd like to do here.